client/src/components/TreeDropdownField/TreeDropdownField.scss

```diff
@@ -96,14 +101,22 @@
   }
 }
 
-.is-focused:not(.is-open)>.Select-control,
-.is-open>.Select-control,
-.Select-menu-outer {
+.Select-arrow-zone {
+  vertical-align: initial;
```

Comment:
This seems to actually just move the arrow up by about 1 pixel instead of vertically centering it.

Reply:
I'm not sure about this? See screenshot below

Reply:
![image](https://user-images.githubusercontent.com/541886/116501068-03951f80-a904-11eb-8947-a4a2beae7961.png)

This is what it looks like for me.
Left is the current look.
Right is with vertical-align: initial; applied.

Looks to me like we don't need to do anything here.

Reply:
Have removed the vertical align